XML/TestCases/FunctionalTests-KernelDebug.xml

```diff
@@ -0,0 +1,15 @@
+<TestCases>
+    <test>
+        <testName>KERNEL-DEBUG-BASIC</testName>
+        <testScript>KERNEL-DEBUG-BASIC.ps1</testScript>
+        <files>.\Testscripts\Linux\check_traces.sh,.\Testscripts\Linux\utils.sh</files>
+        <Platform>Azure,HyperV</Platform>
+        <Category>Functional</Category>
+        <Area>CORE</Area>
+        <Tags>boot</Tags>
+        <Priority>1</Priority>
```

Comment:
Can you make this TestCase applicable for other distro?  if not available for other distro, how can it be Priority 1?  @LiliDeng   what's your opinion?

Reply:
@harz566, @LiliDeng  this case test scenario - checking "kernel debug" is suitable for all the distributions, but I do not have other distro environment and not quite familiar with other distro whether can use "grub2-set-default 0" to select debug kernel.  If someone familiar with othe distro, they could update it. 

For current status, I can update the case title with RHEL- and priority if you prefer to change them.

Thank you.

Reply:
@lixuemin2016 Thank you if you help make the change. Maybe we can change the Priority to 2 for now, leave the TEST Name not changed. Will see if it can be adopt by other distros later.

Reply:
@harz566 updated Priority to 2, thank you so much.